ui/overviewTab/overviewTab.scss

```diff
@@ -13,12 +13,19 @@
     display: flex;
     gap: 16px;
   }
-
+  
   .advantage-overview {
     $advantage-grid-template: getResponsiveGridWidths(128px);
     @include epic-table-grid('advantage', $advantage-grid-template);
     @include epic-table-grid('extraadvantage', $advantage-grid-template);
 
+    input.advantageField:not([value])~.advantage-row,
+    input.advantageField[value=""]~.advantage-row,
+    input.advantageField[value=" "]~.advantage-row {
```

Comment:
It looks like if you choose a blank select option, it gets set to a space...
The first and third options are needed to support initial and the blank select option.
The second selector may not be needed, but is there for safety...